checker/errors/errors.go

```diff
@@ -23,20 +23,27 @@ type TODO struct {
 	filename string
 	lines    []string
 	linecnt  int
+	issueId  string
```

Comment:
This should be a `metadata map[string]string` field similar to the json output struct

Reply:
Changed according to suggestion